statsd/tests.py

```diff
@@ -868,21 +868,20 @@ def test_pipeline_negative_absolute_gauge_tcp():
 
 def _test_big_numbers(cl, proto):
     num = 1234568901234
-    result = 'foo:1234568901234|%s'
     tests = (
         # Explicitly create strings so we avoid the bug we're trying to test.
-        ('gauge', 'g'),
-        ('incr', 'c'),
-        ('timing', 'ms'),
+        ('gauge', 'g', 'foo:1234568901234|%s'),
```

Comment:
might as well just drop the `suffix` part and make the `result` the complete thing (along with adjusting the `_check` method to skip the `result % suffix` line, which is the only place `suffix` gets used.